initializer_inputs/industry/industry_aluminium_production.ad

```diff
@@ -0,0 +1,3 @@
+- query = UPDATE_WITH_FACTOR(V(industry_aluminium_production), preset_demand, USER_INPUT())
+- priority = 10
+- update_type = %
```

Comment:
If we change the %y to absolute, then maybe also the % to absolute!?
But please talk to @AlexanderWirtz about this. Maybe we do not even need the update_type anymore for init. inputs?
I guess for new datasets/scenarios that would be desirable. For the old ones it might make the migration a lot more complicated though (see #1229).